pkg/codegen/internal/test/testdata/types.json

```diff
@@ -421,6 +484,43 @@
         "requiredNumber",
         "requiredString"
       ]
+    },
+    "typetests:regressions:pulumi/pulumi/7454": {
+      "description": "Regression test for pulumi/pulumi#7454",
+      "properties": {
+        "nestedArray": {
+          "type": "array",
+          "items": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "language": {
+            "test": {
+              "expected": {
+                "dotnet": {
+                  "input": "InputList<ImmutableArray<string>>",
+                  "plain": "ImmutableArray<ImmutableArray<string>>"
+                },
+                "go": {
+                  "input": "pulumi.StringArrayArrayInput",
```

Comment:
I might need a build a reference for these for myself :) I dread remembering which type of values this input carries.

Reply:
Yup. Can't wait for the day that this is `pulumi.Input[[]pulumi.Input[[]pulumi.Input[string]]]`

Reply:
Ok so this is consistent with node+py. The .NET is the outlier 🧠 💥 here. I will follow up on that.